### DISCUSSION TITLE
Enabling playground access to users

### DISCUSSION THREAD

**Discussion by orlevii** [open-webui/open-webui#3572] [Ideas]
**Is your feature request related to a problem? Please describe.**
Users can't access the playground, they get redirected back to `/`.
That happens because of this line:
https://github.com/open-webui/open-webui/blob/main/src/lib/components/playground/Playground.svelte#L215

**Describe the solution you'd like**
Allow users to access the playground.

**Describe alternatives you've considered**
* Option 1 - By removing condition mentioned in the code above
* Option 2 - By setting a toggle in Admin menu that permits users to access the playground
* Option 3 - Any other alternative you prefer

----

I would like to better understand the reasoning behind the current implementation

**Comment by smartynov** [comment]
Any updates on this? That would be helpful to have, probably as a separate permission checkbox (as for knowledge, etc)

**Comment by Classic298** [comment]
https://github.com/open-webui/open-webui/pull/13480 created a pr

**Reply by lazariv** [reply]
As I mentioned in that PR, it is a feature that should not be limited to admin users only. Especially according to the [documentation](https://docs.openwebui.com/features/#-model-management) further below, playground is described as a feature:

> 🎲 Model Playground (Beta): Try out models with the model playground area (beta), which enables users to test and explore model capabilities and parameters with ease in a sandbox environment before deployment in a live chat environment. Export your playground conversations in JSON format (compatible with Open WebUI chat import) or as plain text for easy sharing and documentation.
